subsys/bluetooth/controller/include/ll.h

```diff
@@ -38,6 +38,18 @@ u32_t ll_wl_clear(void);
 u32_t ll_wl_add(bt_addr_le_t *addr);
 u32_t ll_wl_remove(bt_addr_le_t *addr);
 
+#if defined(CONFIG_BLUETOOTH_CONTROLLER_PRIVACY)
+u32_t ll_rl_size_get(void);
+u32_t ll_rl_clear(void);
+u32_t ll_rl_add(bt_addr_le_t *id_addr, const u8_t pirk[16],
+		const u8_t lirk[16]);
+u32_t ll_rl_remove(bt_addr_le_t *id_addr);
+u32_t ll_rl_prpa_get(bt_addr_le_t *id_addr, bt_addr_t *prpa);
+u32_t ll_rl_lrpa_get(bt_addr_le_t *id_addr, bt_addr_t *lrpa);
+u32_t ll_rl_enable(u8_t enable);
+void  ll_rl_timeout_set(u16_t timeout);
+#endif
```

Comment:
In most other places we haven't bothered to add #ifdef guards for function prototypes in header files, since having them always around does no harm as long as no one is trying to call them (in which case linking would fail), i.e. it should be sufficient to have the guards only where calling the functions. Another reason for not having the guards in the header file is that otherwise the calls to the function's can't be behind IS_ENABLED(). Code behind this convenience macro still goes through the syntax checks and if the compiler can't see the prototype of a function it'll generate a warning or error (even though the function would never show up in the final image).